kit/scrolls/android/src/main/java/tonlabs/uikit/scrolls/UIKitAccordionOverlayView.java

```diff
@@ -0,0 +1,356 @@
+    }
+
+    // MARK:- Helpers
+
+    private void disableScrollViewIfAny() {
+        View view = getChildAt(0);
+
+        if (!(view instanceof ReactScrollView)) {
+            return;
+        }
+
+        ((ReactScrollView)view).setScrollEnabled(false);
+    }
+
+    private void enableScrollViewIfAny() {
+        View view = getChildAt(0);
+
+        if (!(view instanceof ReactScrollView)) {
+            return;
+        }
+
+        ((ReactScrollView)view).setScrollEnabled(true);
+    }
+
+    private void listenToScrollChangesIfAny() {
+        View view = getChildAt(0);
+
+        if (!(view instanceof ReactScrollView)) {
+            return;
+        }
+
+        ReactScrollViewHelper.addScrollListener(mScrollListener);
+        prevScrollY = view.getScrollY();
+    }
+
+    private void unlistenToScrollChangesIfAny() {
+        View view = getChildAt(0);
+
+        if (!(view instanceof ReactScrollView)) {
+            return;
+        }
+
+        ReactScrollViewHelper.removeScrollListener(mScrollListener);
+        prevScrollY = 0;
+    }
+
+    private void adjustContainerPositionWhenScrollHappened(float shift) {
+        float currentPosition = mOverlayContainer.getTranslationY();
+        mOverlayContainer.setTranslationY(currentPosition + shift);
+    }
+
+    private int getImageTop(int startY) {
+        if (startY == 0) {
+            return 0;
+        }
+
+        View view = getChildAt(0);
+
+        if (view instanceof ReactScrollView) {
+            return startY - ((ReactScrollView) view).getScrollY();
+        }
+        return startY;
+    }
+
+    // MARK:- screenshot
+
+    private BitmapDrawable takeScreenshot(int startY, int endY) {
+        View view = getChildAt(0);
+
+        if (view instanceof ReactScrollView) {
+            return takeScrollViewScreenshot((ReactScrollView) view, startY, endY);
+        }
+        return takeScreenshot(view, startY, endY);
+    }
+
+    private BitmapDrawable takeScreenshot(View view, int startY, int endY) {
+        return takeScreenshot(view, startY, endY, view.getWidth(), view.getHeight());
+    }
+
+    private BitmapDrawable takeScrollViewScreenshot(ReactScrollView scrollView, int startY, int endY) {
+        int width = scrollView.getWidth();
+        int height = scrollView.getChildAt(0).getHeight();
+
+        return takeScreenshot(scrollView.getChildAt(0), startY, endY, width, height);
+    }
+
+    private BitmapDrawable takeScreenshot(View view, int startY, int endY, int width, int height) {
+        int lHeight = Math.max(height, getHeight());
+        // TODO: might be a good idea to re-use from pool
+        // may use - https://github.com/facebook/fresco/blob/a5456ad2cdb9214ebefaf11128d18e1c9edc7784/imagepipeline/src/main/java/com/facebook/imagepipeline/memory/DummyTrackingInUseBitmapPool.java
+        Bitmap fullBitmap = Bitmap.createBitmap(width, lHeight, Bitmap.Config.ARGB_8888);
+
+        final Paint paint = new Paint();
+        // TODO
```

Comment:
todo what?)

Reply:
Filling the transparent pixels with a color :)

Reply:
For now it doesn't work with the theme switching